modules/ROOT/partials/configuration/link_default_protocol.adoc

```diff
@@ -3,6 +3,7 @@
 
 This option allows you to set a default protocol for links when inserting/editing a link via the link dialog. The protocol will apply to any links where the protocol has not been specified and the prefix prompt has been accepted.
 
+// TODO: Should this be shown if already on the autolink page?
```

Comment:
- [ ] Log a Jira for this and remove the TODO

Reply:
Jira has been logged: DOC-1608